app/gui2/src/components/SvgIcon.vue

```diff
@@ -1,4 +1,10 @@
 <script setup lang="ts">
+/**
+ * A component displaying svg icon.
+ *
+ * It displays one group defined in `@/assets/icons.svg` file, specified by `variant` property.
+ *
```

Comment:
trailing blank line - might not be disallowed but might as well be consistent, i guess